package.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Please see https://github.com/ciena-blueplanet/pr-bumper/blob/master/package.json for the proper way to set up this file